java/code/src/com/redhat/rhn/taskomatic/TaskoXmlRpcHandler.java

```diff
@@ -551,11 +547,14 @@ public List<TaskoSchedule> reinitializeAllSchedulesFromNow() {
         List<TaskoSchedule> schedules = new ArrayList<>();
         Date now = new Date();
         for (TaskoSchedule schedule : TaskoFactory.listFuture()) {
-            TaskoSchedule reinited =
-                    TaskoFactory.reinitializeScheduleFromNow(schedule, now);
-            if (reinited != null) {
+            try {
+                TaskoSchedule reinited = TaskoFactory.reinitializeScheduleFromNow(schedule, now);
                 schedules.add(reinited);
             }
+            catch (InvalidParamException | SchedulerException e) {
+                log.error("Unable to reinitialize schedule for job {}", schedule.getJobLabel(), e);
+            }
```

Comment:
Seems like I doubled the curly brace with my suggestion, sorry :man_facepalming: 
```suggestion
```